contrib/localnet/orchestrator/start-zetae2e.sh

```diff
@@ -319,39 +336,41 @@ if [ "$LOCALNET_MODE" == "upgrade" ]; then
   # Run zetae2e, if the upgrade height is greater than 100 to populate the state
   if [ "$UPGRADE_HEIGHT" -gt 100 ]; then
     echo "Running E2E command to setup the networks and populate the state..."
+      # Use light flag to ensure tests can complete before the upgrade height
+      # skip-bitcoin-dust-withdraw flag can be removed after v23 is released
```

Comment:
can we remove `# skip-bitcoin-dust-withdraw flag can be removed after v23 is released`? looks an outdated comment from v23